keras_cv/losses/ciou_loss_test.py

```diff
@@ -65,21 +66,38 @@ def test_output_shape_relative_formats(self):
         self.assertAllEqual(ciou_loss(y_true, y_pred).shape, ())
 
     def test_output_value(self):
-        y_true = [
-            [0, 0, 1, 1],
-            [0, 0, 2, 3],
-            [4, 5, 3, 6],
-            [2, 2, 3, 3],
-        ]
-
-        y_pred = [
-            [0, 0, 5, 6],
-            [0, 0, 7, 3],
-            [4, 5, 5, 6],
-            [2, 1, 3, 3],
-        ]
-
-        ciou_loss = CIoULoss(bounding_box_format="xyxy")
-
-        # expected value for these values is 1.0320253
-        self.assertAllClose(ciou_loss(y_true, y_pred), 1.0320253)
+        def test_loss_format(bounding_box_format):
```

Comment:
Instead of making a sub-function, let's use a `Parameterized` test (see example [here](https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/object_detection/yolo_v8/yolo_v8_detector_test.py#L113)) for this